数値リテラルの型の曖昧性による rustc エラー

## エラーが発生するコード
```
*a
'
.$# 3.14.floor

%exit
0
```
## エラー内容
```
error[E0689]: can't call method `floor` on ambiguous numeric type `{float}`
   --> /var/folders/fx/qb2gk4n14gdggb_snvqt7tdc0000gn/T/.tmpXGbD96.rs:104:48
    |
104 |             buf.push(ResourceType::Int(((3.14).floor()as i32)));
    |                                                ^^^^^
    |
help: you must specify a concrete type for this numeric value, like `f32`
    |
104 |             buf.push(ResourceType::Int(((3.14_f32).floor()as i32)));
    |                                          ~~~~~~~~
```

--- a/src/code_generator.rs
+++ b/src/code_generator.rs
@@ -964,11 +964,11 @@ fn generate_expr(
 ) -> io::Result<Type> {
     match expr {
         ast::ExprAST::Number(i) => {
-            write!(f, "({})", i)?; // 括弧を外すと負数で無効トークン `<-` が生成される危険あり
+            write!(f, "({}{})", i, Type::actual(&Type::Int))?;
             Ok(Type::Int)
         }
         ast::ExprAST::Double(d) => {
-            write!(f, "({:?})", d)?; // 括弧を外すと負数で無効トークン `<-` が生成される危険あり
+            write!(f, "({:?}{})", d, Type::actual(&Type::Double))?;
             Ok(Type::Double)
         }
         ast::ExprAST::Str(s) => {

